tests/integration/fileTypeTests.py

```diff
@@ -101,11 +101,14 @@ def load_definitions(interfaces, force_tas_load):
         SchemaLoader.loadAllFromPath(join(CONFIG_BROKER["path"],"dataactvalidator","config"))
         SQLLoader.loadSql("sqlRules.csv")
         # Load domain values tables
-        loadDomainValues(join(CONFIG_BROKER["path"],"dataactvalidator","config"),join(CONFIG_BROKER["path"],"tests","sf_133.csv"),join(CONFIG_BROKER["path"],"tests","program_activity.csv"))
+        loadDomainValues(
+            join(CONFIG_BROKER["path"], "dataactvalidator", "config"),
+            join(CONFIG_BROKER["path"], os.path.join("tests", "integration", "data"), "sf_133.csv"),
+            join(CONFIG_BROKER["path"], os.path.join("tests", "integration", "data"), "program_activity.csv"))
```

Comment:
I didn't change this portion, Becky had done this but it looks like first join handles the basePath parameter in loadDomainValues while the others lead to a specific test file that's in a completely different directory